docs/concepts/resource-manager.md

```diff
@@ -434,3 +434,44 @@ serviceaccount.resources.gardener.cloud/skip-deletion: "true"
 ```
 
 then the respective `ServiceAccount` will not be deleted when the `Secret` is deleted.
+
+## Webhooks
+
+### Auto-Mounting Projected `ServiceAccount` Tokens
+
+When this webhook is activated then it automatically injects projected `ServiceAccount` token volumes into `Pod`s and all its containers if all of the following preconditions are fulfilled:
+
+1. The `Pod` is NOT annotated with `projected-token-mount.resources.gardener.cloud/skip=true`.
+2. The `Pod`'s `.spec.serviceAccountName` field is NOT empty and NOT set to `default`.
+3. The `ServiceAccount` specified in the `Pod`'s `.spec.serviceAccountName` sets `.automountServiceAccountToken=false`.
+4. The `Pod`'s `.spec.volumes[]` DO NOT already contain a volume with a name prefixed with `kube-api-access-`.
+
+The projected volume will look as follows:
+
+```yaml
+spec:
+  volumes:
+  - name: kube-api-access-gardener
+    projected:
+      defaultMode: 420
+      sources:
+      - serviceAccountToken:
+          expirationSeconds: 43200
+          path: token
+      - configMap:
+          items:
+          - key: ca.crt
+            path: ca.crt
+          name: kube-root-ca.crt
+      - downwardAPI:
+          items:
+          - fieldRef:
+              apiVersion: v1
+              fieldPath: metadata.namespace
+            path: namespace
+```
+
+> The `expirationSeconds` are defaulted to `12h` and can be overwritten with the `--projected-token-mount-expiration-seconds` flag, or with the `projected-token-mount.resources.gardener.cloud/expiration-seconds` annotation on a `Pod` resource.
```

Comment:
In the other PR (tokenRequestor) we thought about setting an upper limit of 24 hours on validity. Is this reasonable here too?

Reply:
We didn't set such upper limit for the expiration duration of tokens. What we did was to set an upper limit of `24h` for the renew duration only, i.e., tokens valid for 1 month would still be renewed after 24 hours (instead of when 80% of the validity is reached).

The kubelet is already doing the same (from https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/#service-account-token-volume-projection):

>  The kubelet proactively rotates the token if it is older than 80% of its total TTL, or if the token is older than 24 hours.

Reply:
got it. Thanks for the clarification.